### PR TITLE
Changed width on Sidebar.tsx too 100

### PR DESCRIPTION
**Pull request by mkde93**

### CODE REVIEW

File: subhive/src/bars/Sidebar.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Sidebar extends React.Component {
         return (
             <div
                 className={style({
-                    width: 200,
+                    width: 100,
```
Comment (Souzadk) [inline subhive/src/bars/Sidebar.tsx:10]: Can you please not

**Review by Souzadk — CHANGES_REQUESTED** [review]
Check my comment

File: subhive/src/bars/Sidebar.tsx (hunk 1 above)
Comment (mkde93) [reply subhive/src/bars/Sidebar.tsx:10]: it is neccessary that we do tihs

Comment (Souzadk) [reply subhive/src/bars/Sidebar.tsx:10]: Okay bruder

**Review by Souzadk — APPROVED** [review]

**Pull request merged**